osgar/drivers/realsense.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, config, bus):
             self.default_depth_resolution = config.get("depth_resolution", [640, 360])
             self.default_rgb_resolution = config.get("rgb_resolution", [640, 360])
             self.depth_fps = config.get("depth_fps", 30)
+            self.disable_emitor = config.get("disable_emitor", False)
```

Comment:
what is the motivation behind this configuration parameter?

Reply:
Oh, it is the IR emitter, i.e. potential collision of two devices?

Reply:
Yes, that is the motivation.